common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1438,7 +1408,7 @@ def forum_posts_allowed(self):
 
     @property
     def number(self):
```

Comment:
We had a bunch of comments up top for "def number_for_course_location" about the "number". I think either adding some doc about what this "number" is would be helpful, or maybe calling it `course_name` as @doctoryes suggested.